src/olympia/users/tests/test_admin.py

```diff
@@ -477,6 +481,24 @@ def test_can_edit_with_users_edit_permission(self):
         assert alog.arguments == [self.user]
         assert alog.details == {'username': [old_username, 'foo']}
 
+    def test_queries_detail(self):
+        user = user_factory(email='someone@mozilla.com')
+        # We want to see absolutely everything, so make our user a superadmin.
+        self.grant_permission(user, '*:*')
+        self.client.force_login(user)
+        with self.assertNumQueries(21):
+            # - 4 savepoint/release
+            # - 2 current logged in user & groups
+            # - 2 target user & groups
+            # - 1 banned user content instance
+            # - 8 related content counts
```

Comment:
What are these 8 queries - can you include which content we're currently expecting so if it changes it'll be clearer what's new?

Reply:
https://github.com/mozilla/addons-server/pull/22500/commits/a96944c88552cd49d7d375716b8a547bf18c2429